File: src/main/java-graphql/io/github/graphql/model/ConvertProjectCardNoteToIssuePayloadTO.java
package io.github.graphql.model;

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLRequestSerializer;
import java.util.StringJoiner;

@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-06T19:47:48+0800"
)
public class ConvertProjectCardNoteToIssuePayloadTO implements java.io.Serializable {

    private String clientMutationId;
    private ProjectCardTO projectCard;

    public ConvertProjectCardNoteToIssuePayloadTO() {
    }

    public ConvertProjectCardNoteToIssuePayloadTO(String clientMutationId, ProjectCardTO projectCard) {
        this.clientMutationId = clientMutationId;
        this.projectCard = projectCard;
    }

    public String getClientMutationId() {
        return clientMutationId;
    }

    public ProjectCardTO getProjectCard() {
        return projectCard;
    }


    @Override
    public String toString() {
        StringJoiner joiner = new StringJoiner(", ", "{ ", " }");
        if (clientMutationId != null) {
            joiner.add("clientMutationId: " + GraphQLRequestSerializer.getEntry(clientMutationId));
        }
        if (projectCard != null) {
            joiner.add("projectCard: " + GraphQLRequestSerializer.getEntry(projectCard));
        }
        return joiner.toString();
    }

    public static ConvertProjectCardNoteToIssuePayloadTO.Builder builder() {
        return new ConvertProjectCardNoteToIssuePayloadTO.Builder();
    }

    public static class Builder {

        private String clientMutationId;
        private ProjectCardTO projectCard;

        public Builder() {
        }

        public Builder setClientMutationId(String clientMutationId) {
            this.clientMutationId = clientMutationId;
            return this;
        }

        public Builder setProjectCard(ProjectCardTO projectCard) {
            this.projectCard = projectCard;
            return this;
        }


        public ConvertProjectCardNoteToIssuePayloadTO build() {
            return new ConvertProjectCardNoteToIssuePayloadTO(clientMutationId, projectCard);
        }

    }
}

File: src/main/kotlin-graphql/io/github/graphql/model/ConvertProjectCardNoteToIssuePayloadTO.kt
package io.github.graphql.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLRequestSerializer
import java.util.StringJoiner

@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-06T19:48:04+0800"
)
open class ConvertProjectCardNoteToIssuePayloadTO(
    val clientMutationId: String?,
    val projectCard: ProjectCardTO?
) {

    companion object {
        fun builder(): Builder = Builder()
    }

    // In the future, it maybe change.
    override fun toString(): String {
        val joiner = StringJoiner(", ", "{ ", " }")
        if (clientMutationId != null) {
            joiner.add("clientMutationId: " + GraphQLRequestSerializer.getEntry(clientMutationId))
        }
        if (projectCard != null) {
            joiner.add("projectCard: " + GraphQLRequestSerializer.getEntry(projectCard))
        }
        return joiner.toString()
    }

    class Builder {

        private var clientMutationId: String? = null
        private var projectCard: ProjectCardTO? = null

        fun setClientMutationId(clientMutationId: String?): Builder {
            this.clientMutationId = clientMutationId
            return this
        }

        fun setProjectCard(projectCard: ProjectCardTO?): Builder {
            this.projectCard = projectCard
            return this
        }

        fun build(): ConvertProjectCardNoteToIssuePayloadTO {
            return ConvertProjectCardNoteToIssuePayloadTO(clientMutationId, projectCard)
        }
    }
}
